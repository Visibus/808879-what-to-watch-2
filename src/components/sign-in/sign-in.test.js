import SignIn from './sign-in';
import {BrowserRouter} from 'react-router-dom';

it(`SignIn correctly renders after relaunch`, () => {
  const tree = window.renderer.create(
      <BrowserRouter><SignIn
        userEmail={``}
        userPassword={``}
        onChangeUserEmailHandler={jest.fn()}
        onChangeUserPasswordHandler={jest.fn()}
        onSubmitSignIn={jest.fn()}
        isAuthorizationRequired={false}
      /></BrowserRouter>).toJSON();

  expect(tree).toMatchSnapshot();
});
