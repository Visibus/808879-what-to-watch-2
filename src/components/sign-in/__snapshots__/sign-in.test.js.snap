// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignIn correctly renders after relaunch 1`] = `null`;
